spec/google_maps_service/apis/roads_spec.rb

```diff
@@ -17,6 +17,20 @@
     end
   end
 
+  context '#nearest_roads' do
+    before(:example) do
+      stub_request(:get, /https:\/\/roads.googleapis.com\/v1\/nearestRoads/)
+              .to_return(:status => 200, headers: { 'Content-Type' => 'application/json' }, body: '{"snappedPoints":["foo"]}')
+    end
+
+    context 'match' do
```

Comment:
Umm, `match` is also not suitable as context name here, cmiiw. I think the context name can be `single point`.